src/timeline.js

```diff
@@ -51,15 +59,15 @@ export default class Timeline extends React.Component {
     groupOffset: PropTypes.number.isRequired,
     rowLayers: PropTypes.arrayOf(
       PropTypes.shape({
-        start: PropTypes.object.isRequired,
-        end: PropTypes.object.isRequired,
+        start: PropTypes.oneOfType([PropTypes.object, PropTypes.number]).isRequired,
```

Comment:
what happens at runtime, if you feed into this field: a string; or nothing? Does React perform a validation and outputs something? I hope so. Because otherwise, why should we bother to specify things here.

If yes, I'd add constraints also on "items".